Code/Legacy/CrySystem/LevelSystem/SpawnableLevelSystem.cpp

```diff
@@ -82,6 +82,7 @@ namespace LegacyLevelSystem
         }
 
         AzFramework::RootSpawnableNotificationBus::Handler::BusConnect();
+        AzFramework::LevelSystemLifecycleRequestBus::Handler::BusConnect();
```

Comment:
This should instead register itself with an AZ::Interface.
```suggestion
        auto registerOutcome = AzFramework::LevelSystemLifecycleInterface::Register(this);
        AZ_Error("SpawnableLevelSystem", registerOutcome, "Failed to register the SpawnableLevelSystem with the LevelSystemLifecycleInterface: %s", registerOutcome.GetError().c_str());
```

Reply:
Fixed using Interface now! Deriving from Interface<>::Registrar seems fine too, or should I explicitly keep the error checking?

Reply:
I think either work fine, though I would prefer the error message path.

Reply:
Good idea. Keep the RAII, and the error.